src/main/java/net/minecraftforge/common/IHorseArmor.java

```diff
@@ -0,0 +1,22 @@
+package net.minecraftforge.common;
+
+import net.minecraft.entity.passive.EntityHorse;
+import net.minecraft.entity.passive.HorseArmorType;
+import net.minecraft.item.ItemStack;
+import net.minecraft.world.World;
+
+public interface IHorseArmor 
+{
+    /** 
+     * @param stack - The armor stack
+     * @return The HorseArmorType of the armor stack 
```

Comment:
`@return` follows the same casing and punctuation conventions as `@param`. Also, please wrap any Java identifiers in `{@code}`. [Reference (style guide)](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#styleguide) [Reference (`@return`)](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#@return)